templates/_helpers.tpl

```diff
@@ -111,6 +111,10 @@ Insert LDAP environment variables
 */}}
 {{- define "georchestra.ldap-envs" -}}
 {{- $ldap := .Values.ldap -}}
+{{- $ldap_secret_georchestra_name := printf "%s-ldap-passwords-secret" (include "georchestra.fullname" .) -}}
+{{- if $ldap.existingSecret }}
+{{- $ldap_secret_georchestra_name = $ldap.existingSecret -}}
```

Comment:
can't this be refactored with a "default" function from helm?
similar to https://github.com/georchestra/helm-georchestra/blob/41289eccb30e4e70ac800ddbee6c433b41c56cd4/templates/_helpers.tpl#L5

Reply:
Quite possibly. I must admit I merely copied the logic from https://github.com/georchestra/helm-georchestra/blob/main/templates/_helpers.tpl#L71

Reply:
OK, it should be better now